README.md

```diff
@@ -17,8 +17,8 @@ Usage
 ```js
 var voidElements = require('void-elements');
 
-assert(voidElements.indexOf('span') === -1, '<span> is not a void element');
-assert(voidElements.indexOf('img') !== -1, '<img> is a void element');
+assert(voidElements.span !== true, '<span> is not a void element');
+assert(voidElements.img === true, '<img> is a void element');
```

Comment:
It would probably be more helpful for people trying to use the module if we did `voidElement['span']` rather than `voidElements.span` so it's a little more obvious that you can still use variables etc. and look up whether they are void elements.